### ISSUE TITLE
There are references to non-existent shims dir that needs to be updated

### ISSUE THREAD

**Issue by razajafri** [NVIDIA/spark-rapids#4920]
```
git grep '[^/]shims/'
CONTRIBUTING.md:There is a known issue with the shims/spark3xx submodules. After being enabled once, a module such as shims/spark312
docs/dev/shims.md:(i.e., in `sql-plugin/src/main/scala` as opposed to be duplicated either in `shims/spark3xx` submodules or over
jenkins/Jenkinsfile-blossom.premerge:                                                                      'sql-plugin/src/main/scala/,shims/spark311/src/main/scala/,' +
jenkins/Jenkinsfile-blossom.premerge:                                                                      'shims/spark301db/src/main/scala/,shims/spark301/src/main/scala/,' +
jenkins/Jenkinsfile-blossom.premerge:                                                                      'shims/spark302/src/main/scala/,shims/spark303/src/main/scala/,' +
jenkins/Jenkinsfile-blossom.premerge:                                                                      'shims/spark304/src/main/scala/,shims/spark312/src/main/scala/,' +
jenkins/Jenkinsfile-blossom.premerge:                                                                      'shims/spark313/src/main/scala/'
```

_Originally posted by @gerashegalov in https://github.com/NVIDIA/spark-rapids/issues/4857#issuecomment-1062160432_

**Comment by razajafri** [comment]
@pxLi is assigned to an issue #4991 which should handle the references to non-existent `shims/spark*` folders. I will address the changes in `.md` file

**Comment by razajafri** [comment]
I am limiting this issue to only address the references in `.md` files 

The jacoco references in Jenkinsfile-blossom.premerge will be addressed by #4991

**Comment by pxLi** [comment]
https://github.com/NVIDIA/spark-rapids/pull/5067
